### DISCUSSION TITLE
Where is the Wacom ArtPad and Wacom SD-510C mouse option?

### DISCUSSION THREAD

**Discussion by ghost** [86Box/86Box#3876] [Q&A] [answered]
Hello there, and this is my first 86Box Q&A question.

In the Essentials 86Box website, I found a driver under the (\drivers\input) folder, that has a Wacom ArtPad driver called (Wacom ArtPad (Windows NT4, 98, 2000, XP).zip), and in v4.1 build 5226, it doesn't even show both the Wacom ArtPad and Wacom SD-510C mouse tablet devices, but on build 4792, it shows up the Wacom mouse tablet devices, Here is the result in build 5226/4792:
(5226 in the top, 4792 in the bottom.)
![image](https://github.com/86Box/86Box/assets/152977043/b9871fd2-e543-4b6c-a7f9-e01bde120d94)
![image](https://github.com/86Box/86Box/assets/152977043/6d265118-23b9-47f6-af9b-762892f3ae64)

I ran into some tests in my Windows 98 SE machine and turned out the Wacom ArtPad driver doesn't even work on the mouse tablet device (on build 4792):
![Monitor_1_20231211-184849-648](https://github.com/86Box/86Box/assets/152977043/ad58b5f6-d238-46c6-a4a7-a869264cb8ec)
![Monitor_1_20231211-185202-553](https://github.com/86Box/86Box/assets/152977043/d9e0e808-9d40-431c-87e6-6f87b3fe4e2c)

Here is the result in (essentials.86box.net):
![image](https://github.com/86Box/86Box/assets/152977043/4b3956e2-10df-471a-b2f7-c6456ed9abad)

**Comment by jriwanek** [comment, marked as answer]
It's been disabled for the time as it's incomplete.

Also, you really don't need to tell us the numeric order of each and every issue and discussion you open.

**Reply by ghost** [reply]
Oh okay...
